components/ProductCard/index.js

```diff
@@ -3,14 +3,19 @@ import styled from "styled-components";
 import { formatToEUR } from "../../helper/formatToEUR";
 import { useRouter } from "next/router";
 import useStore from "../../hooks/useStore";
+import "chart.js/auto";
+import { Bar } from "react-chartjs-2";
+import { chartOptions } from "../../helper/chartOptions";
+import { createChartData } from "../../helper/createChartData";
+import useMedia from "../../hooks/useMedia";
+import { sumTotalPrice } from "../../helper/sumTotalPrice";
 
 export default function ProductCard({ project, editState }) {
   const router = useRouter();
   const [moveToActive] = useStore((state) => [state.moveToActive]);
-  const sumUpArray = (accumulator, currentValue) => accumulator + currentValue;
-  const totalPrice = project?.items
-    .map((item) => Number(item.price))
-    .reduce(sumUpArray, 0);
+  const labelColor = useMedia(["(prefers-color-scheme: dark)"], [true], false)
+    ? "white"
+    : "black";
 
   if (project === undefined) {
```

Comment:
Is this only useful for `undefined` or for all falsy values?
```suggestion
  if (!project) {
```

Reply:
yeah you're correct, i can check every falsy values